src/widgets/ReportTable/ReportCell.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import { APP_FONT_FAMILY, BACKGROUND_COLOR } from '../../globalStyles';
+
+export const ReportCell = ({ even, children }) => {
+  const backgroundColor = even ? { backgroundColor: 'white' } : null;
+  return (
+    <View style={[localStyles.container, backgroundColor]}>
+      <Text style={localStyles.cell}>{children}</Text>
+    </View>
+  );
+};
+
+ReportCell.propTypes = {
+  even: PropTypes.bool,
+  children: PropTypes.string.isRequired,
+};
+
+const localStyles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    paddingLeft: '2%',
```

Comment:
probably better to not use this and just use flexbox .. but we can organise styles later when the component is all together.. much easier!

Reply:
Agree, we can work on it here: #1751 
Thanks!